src/Admin/FieldDescriptionCollection.php

```diff
@@ -19,15 +19,22 @@
 final class FieldDescriptionCollection implements \ArrayAccess, \Countable
 {
     /**
-     * @var FieldDescriptionInterface[]
+     * @var array<string, FieldDescriptionInterface>
      */
     private $elements = [];
 
     public function add(FieldDescriptionInterface $fieldDescription): void
     {
-        $this->elements[$fieldDescription->getName()] = $fieldDescription;
+        $name = $fieldDescription->getName();
+
+        \assert(\is_string($name));
```

Comment:
Yes, because PHPStan was complaining. I'd rather make `FieldDescriptionInterface::getName` return always a string, but I don't think this should be in this PR.

Reply:
Can you leave this one as a phpstan baseline ignore? Maybe it is better to know that we have some work to do here

Reply:
I agree: I'm working on it
https://github.com/sonata-project/SonataAdminBundle/pull/6503